snippets/serializeForm.md

```diff
@@ -0,0 +1,21 @@
+### serializeForm
+
+Serializes a form into an object.
+
+First we transform the `form` into `FormData`, then we convert it into an `array` and from the `array` we collect an `query string`
+
+```js
+const serializeForm = form =>
+  Array.from(new FormData(form), field => field.map(encodeURIComponent).join('=')).join('&')
+```
+
+```html
```

Comment:
Including an `html` example here is not necessary and makes the snippet noisy. It's easy to understand the context from the JS result below.